### DISCUSSION TITLE
Wi-fi operations

### DISCUSSION THREAD

**Discussion by Zephyrxx0** [rafaelvcaetano/melonDS-android#368] [General]
Please implement wifi as it is very helpful and a really important feature. It would be quite revolutionary amonh nds emulators. I know it can be hard but the pc version has one for quite some time now. You can always take some help from it. P.s. Fast Forward opt. not working properly.

**Comment by rafaelvcaetano** [comment]
Android has some restrictions that don't allow the same functionality used in the PC version to be easily ported. I've seen you have commented on [this discussion regarding Wi-Fi](https://github.com/rafaelvcaetano/melonDS-android/discussions/317). You can try to follow the steps the other users have posted for now until a better solution is implemented.

**Comment by ashgx** [comment]
Please don't force developers, OK?If you are Chinese, you have no right to ask developers to speed up the development.I hate people who force developers, especially Chinese like beggars.

**Reply by FlashlightET** [reply]
i get your point but what does being chinese have to do with anything

**Reply by ashgx** [reply]
Because a significant number of Chinese people make demands on the Internet that cannot be fulfilled in a short period of time, or they make demands that are simply unreasonable.Even they harass developers
